Sources/SpeziAccount/AccountValue/Configuration/AccountValueConfiguration.swift

```diff
@@ -58,21 +58,30 @@ public struct AccountValueConfiguration {
         }
     }
 
-    func allCategorized(filteredBy filter: [AccountKeyRequirement]? = nil) -> OrderedDictionary<AccountKeyCategory, [any AccountKey.Type]> {
-        // swiftlint:disable:previous discouraged_optional_collection
-        if let filter {
-            return self.reduce(into: [:]) { result, configuration in
-                guard filter.contains(configuration.requirement) else {
-                    return
-                }
+    func allCategorizedForDisplay(
```

Comment:
This will currently not display any display-only AccountKeys if I'm not mistaken.

Reply:
Tagging @Supereg here.

Reply:
Probably not. Thanks for looking into this at this level of detail. I will need to double check 👍

Reply:
I have rewritten that part a bit. The issue should be resolved for. 

For referee: by specifying the `AccountKeyRequirement` you already implicitly specify the AccountKeyOption as well (if you want to filter for `.required, .collect` you will only get mutable account keys that are available to be displayed). Though this is a very implicit check as part of the initialization function. Therefore, I still included this check at this spot.